### ISSUE TITLE
Option to disable library refresh on startup

### ISSUE THREAD

**Issue by pelupessy** [Heroic-Games-Launcher/HeroicGamesLauncher#3359]
### Problem description

Because the library is not displayed during refresh it ads time to the launcher's startup. Additionally, server-side and/or network problems could potentially block functionality.

### Feature description

Add option "disable library refresh at startup" in the settings or an "offline" toggle in the ui.

### Alternatives

Brute-force the issue by disabling the network during heroic's startup. 

### Additional information

This feature has also been requested at: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/2655, but that issue was closed without the feature having been implemented.

**Comment by arielj** [comment]
just to clarify, heroic already refreshes the library in the background (the library should be visible showing the cached information while it's refreshed)

if it's doing a full foreground refresh there's a different issue going on

**Comment by Sunspark-007** [comment]
I would like to ask about this refresh thing.. does it actually refresh? I have 3 stores added.. and when I launch Heroic I see the arrows spinning, but usually the # doesn't change.. but when they stop spinning and I click on the refresh button to make them spin again, the counter does increment and the new titles get added.

So I think there is an issue.

**Comment by arielj** [comment]
yeah, I noticed that too, the first time it seems to run but does not update the frontend, I haven't had time to debug that though

but that should happen in the background, it should do a complete library foreground refresh, it shouldn't slow down heroic's start up

**Comment by pelupessy** [comment]
It does not look like the library refresh at startup is performed in the background. Alternatively, it is being performed "in the background", but the library only gets displayed after heroic has done whatever it is doing at startup, which happens to include the refresh. 

Regardless, heroic will display the library nearly instantly when starting it without internet enabled. So, if it is another problem that is delaying things, it involves something heroic is doing online.
